lib/memoryjs.cc

```diff
@@ -146,6 +148,60 @@ Napi::Value findModule(const Napi::CallbackInfo& args) {
   }
 }
 
+Napi::Value findPattern(const Napi::CallbackInfo& args) {
+  Napi::Env env = args.Env();
+
+  // Address of findPattern result
+  uintptr_t address = -1;
+
+  // Define error message that may be set by the function that gets the modules
+  const char* errorMessage = "";
+
+  pid_t hProcess = (pid_t)args[0].As<Napi::Number>().Int64Value();
+
+  std::vector<module::Module> moduleEntries = module::getModules(hProcess, &errorMessage);
```

Comment:
I don't know if we are actually crashing here, but out of interest, did you try to use findModule here? since we know it works? and find the pattern on that directly?

Reply:
I haven't tried using `findModule` yet, but I'll see if it helps later today.

Reply:
I just tried `findModule()` instead of using `getModule()`.

I think my `findPattern()` function in `pattern.cc` is not completely working
yet, because CrewLink now exits immediately after it sees Among Us, and
`dmesg` is showing some segfaults for electron.